src/System.Diagnostics.Debug.SymbolReader/src/System/Diagnostics/Debug/SymbolReader/SymbolReader.cs

```diff
@@ -0,0 +1,120 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.Reflection.Metadata;
+using System.Reflection.Metadata.Ecma335;
+using System.Reflection.PortableExecutable;
+
+namespace System.Diagnostics.Debug.SymbolReader
+{
+
+    public class SymbolReader
+    {
+    
+        /// <summary>
+        /// Checks availability of debugging information for given assembly.
+        /// </summary>
+        /// <param name="assemblyName">file name of the assembly</param>
+        /// <returns>true if debugging information is available</returns>
+        public static bool LoadSymbolsForModule(string assemblyName)
+        {
+            MetadataReader peReader, pdbReader;
+            
+            return GetReaders(assemblyName, out peReader, out pdbReader);
+        }
+    
+        /// <summary>
+        /// Returns method token and IL offset for given source line number.
+        /// </summary>
+        /// <param name="assemblyName">file name of the assembly</param>
+        /// <param name="fileName">source file name</param>
+        /// <param name="lineNumber">source line number</param>
+        /// <param name="methToken">method token return</param>
+        /// <param name="ilOffset">IL offset return</param>
+        public static void ResolveSequencePoint(string assemblyName, string fileName, int lineNumber, out int methToken, out int ilOffset)
+        {
+            MetadataReader peReader, pdbReader;
+            methToken = 0;
+            ilOffset = 0;
+            
+            if(!GetReaders(assemblyName, out peReader, out pdbReader))
```

Comment:
Nit: there should be a space between the `if` and first parens. Same with between `foreach` and the first parens below. Applies throughout.